src/gui/MainWindow.cpp

```diff
@@ -431,6 +431,13 @@ void MainWindow::databaseTabChanged(int tabIndex)
 
 void MainWindow::closeEvent(QCloseEvent* event)
 {
+
+  if (config()->get("GUI/CloseToTray").toBool()) {
+    event->ignore();
+    QTimer::singleShot(0, this, SLOT(hide()));
```

Comment:
I've never worked with QT before.  Is the `singleShot` `hide` to release the event thread?